.gitignore

```diff
@@ -2,4 +2,6 @@
 .*.swp
 *.egg-info
 .coverage
-dist/
\ No newline at end of file
+dist/
+
+
```

Comment:
there should be only one line space at the end of the file so remove the second one.